test/model.user-identity.test.js

```diff
@@ -161,4 +161,34 @@ describe('UserIdentity', function () {
       });
   });
 
+  it('supports ldap login', function(done) {
+    var identity = { emails: [
+        {value: 'fooldap@bar.com'}
+      ], id: 'f123ldap', username: 'xyzldap'
+    };
+    var credentials = {accessToken: 'atldap1', refreshToken: 'rtldap1'};
+    var options = {autoLogin: false};
+    UserIdentity.login('ldap', 'ldap', identity, credentials, options,
+      function(err, user, identity, token) {
```

Comment:
if you tab this in two times, you can pull back all the inner items back one indent

...options,
    function(err, user, identity...)
  if (err) return done(err)
```